src/box/lua/config/configdata.lua

```diff
@@ -191,6 +195,25 @@ local function new(iconfig, cconfig, instance_name)
         end
     end
 
+    local bootstrap_strategy = instance_config:get(iconfig_def,
+        'replication.bootstrap_strategy')
+    local bootstrap_leader = found.replicaset.bootstrap_leader
+    if bootstrap_strategy ~= 'config' then
+        if bootstrap_leader ~= nil then
+            error('The "bootstrap_leader" option cannot be set if '..
+                  '"bootstrap_strategy" is not "config"', 0)
+        end
+    elseif bootstrap_leader == nil then
+        error('Incorrect value for option "bootstrap_leader": the option '..
+              'can\'t be empty when bootstrap strategy is "config"', 0)
```

Comment:
Nit: Shouldn't we add instance/replicaset/group names into these error messages like it is done for other error messages above and below?